.pre-commit-config.yaml

```
@@ -0,0 +1,40 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+
+# exclusions should be separated with a pipe (|) character and a newline
+exclude: |
+  (?x)^(
+      tests/expected/.*
+  )$
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v5.0.0
+    hooks:
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-yaml
```

We also have toml files => lets check these as well, or?


```suggestion
      - id: check-toml
      - id: check-yaml
```